docs/tutorials/basic_ua_commands.md

```diff
@@ -0,0 +1,258 @@
+
+Additionally, if you look at the last lines of the output, you can identify that this machine is not
+attached to an UA subscription.
+```
+This machine is not attached to a UA subscription.
+See https://ubuntu.com/advantage
+```
+
+### Attach
+
+To access any of those service offerings, you need to attach UA to a subscription. This is
+achieved by running the attach command. Before you run it, you need to get an UA token.
+Any user with a Ubuntu One account is entitled to a free personal token to use with UA.
+You can retrieve your UA token from the [advantage](https://ubuntu.com/advantage/) portal.
+You will log in with your SSO credentials, the same credentials you use for https://login.ubuntu.com.
+After getting your UA token, go to the LXD container and run:
+
+```console
+$ sudo ua attach YOUR_TOKEN
+```
+
+It is expected for you to see an output similar to this one:
+
+```
+Enabling default service esm-infra
+Updating package lists
+UA Infra: ESM enabled
+This machine is now attached to 'USER ACCOUNT'
+
+SERVICE       ENTITLED  STATUS    DESCRIPTION
+cis           yes       disabled  Center for Internet Security Audit Tools
+esm-infra     yes       enabled   UA Infra: Extended Security Maintenance (ESM)
+fips          yes       n/a       NIST-certified core packages
+fips-updates  yes       n/a       NIST-certified core packages with priority security updates
+livepatch     yes       n/a       Canonical Livepatch service
+
+NOTICES
+Operation in progress: ua attach
+
+Enable services with: ua enable <service>
+
+                Account: USER ACCOUNT
+           Subscription: USER SUBSCRIPTION
+            Valid until: 9999-12-31 00:00:00+00:00
+Technical support level: essential
+```
+
+From this output, you can see that the attach command enables all of the services specified by the user subscription.
+After the command ends, UA display what is the state of the machine now.
+That status output is exactly what you will see if you run the status command again.
+You can confirm this by running:
+
+```console
+$ ua status
+```
+
+One question that may arise is that the output of status and unattached are different. When
+attached, status presents two new columns, **ENTITLED** and **STATUS**, while also dropping the
+**AVAILABLE** column. For more information of why the output is different, please refer to this
+[explanation](../explanations/status_columns.md).
+
+Finally, another useful bit at the end of both attach and status is the contract expiration date:
+
+```
+    Account: USER ACCOUNT
+Subscription: USER SUBSCRIPTION
+Valid until: 9999-12-31 00:00:00+00:00
+```
+
+The `Valid until` field describes when your contract will be expired, so you can be aware of when it
+needs to be renewed.
+
+
+### Refresh
+
+In the last section, we mentioned that your contract can expire. After you renew the contract
+how can you make our machine aware of that ? You can do this through this command:
```

Comment:
We should include some assurance here that free tokens do not expire.

We can say something like "While free tokens never expire, if you buy a UA subscription, and later need to renew the contract, you can make your machine aware of it by running this command."

Reply:
Modified it a little bit, but I agree it is information worth having on the tutorial